website/static/js/_allLogTexts.json

```diff
@@ -0,0 +1 @@
 for ${node}","dropbox_node_deauthorized":"${user} deauthorized the Dropbox addon for ${node}","figshare_content_linked":"${user} linked content from Figshare in ${node}","figshare_content_unlinked":"${user} unlinked content from Figshare in ${node}","figshare_file_added":"${user} added file ${path} to Figshare in ${node}","figshare_file_removed":"${user} removed file ${path} from Figshare in ${node}","figshare_node_authorized":"${user} authorized the Figshare addon from ${node}","figshare_node_deauthorized":"${user} deauthorized the Figshare addon from ${node}","forward_url_changed":"${user} change forward URL to ${forward_url} in ${node}","github_file_added":"${user} added ${path} to GitHub repo ${repo} in ${node}","github_file_removed":"${user} removed ${path} in GitHub repo ${repo} in ${node}","github_file_updated":"${user} updated ${path} in GitHub repo ${repo} in ${node}","github_folder_created":"${user} created folder ${path} in GitHub repo ${repo} in ${node}","github_node_authorized":"${user} authorized the GitHub addon for ${node}","github_node_deauthorized":"${user} deauthorized the GitHub addon for ${node}","github_repo_linked":"${user} linked GitHub repo ${repo} in ${node}","mendeley_folder_selected":"${user} selected ${folder_name} in Mendeley in ${node}","mendeley_node_authorized":"${user} authorized the addon Mendeley for ${node}","mendeley_node_deauthorized":"${user} deauthorized the addon Mendeley for ${node}","zotero_folder_selected":"${user} linked Zotero folder ${folder_name} to ${node}","zotero_node_authorized":"${user} authorized the Zotero addon for ${node}","zotero_node_deauthorized":"${user} deauthorized the Zotero addon for ${node}","s3_bucket_linked":"${user} linked the Amazon S3 bucket ${bucket} to ${node}","s3_bucket_unlinked":"${user} unselected the Amazon S3 bucket ${bucket} in ${node}","s3_file_added":"${user} added file ${path} to Amazon S3 bucket ${bucket} in ${node}","s3_file_removed":"${user} removed ${path} in Amazon S3 bucket ${bucket} in ${node}","s3_file_updated":"${user} updated ${path} in Amazon S3 bucket ${bucket} in ${node}","s3_folder_created":"${user} created folder ${path} in Amazon S3 in ${node}","s3_node_authorized":"${user} authorized the Amazon S3 addon for ${node}","s3_node_deauthorized":"${user} deauthorized the Amazon S3 addon for ${node}","googledrive_file_added":"${user} added file ${googledrive_path} to Google Drive in ${node}","googledrive_file_removed":"${user} removed file ${googledrive_path} from Google Drive in ${node}","googledrive_file_updated":"${user} updated file ${googledrive_path} in Google Drive in ${node}","googledrive_folder_created":"${user} created folder ${googledrive_path} in Google Drive in ${node}","googledrive_folder_selected":"${user} linked Google Drive folder ${googledrive_folder} to ${node}","googledrive_node_authorized":"${user} authorized the Google Drive addon for ${node}","googledrive_node_deauthorized":"${user} deauthorized the Google Drive addon for ${node}"}
```

Comment:
I'm fairly certain this is a generated file and should not be in version control.
@samchrisinger might know?